docs-src/content/protobuf/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Cubic API Reference"
-weight: 10
+weight: 3
```

Comment:
Just curious--why are you changing the weights?

Reply:
Probably to reorder pages; it's hard to do some of the reviews here because the HTML would have been so much nicer to look at :) (disadvantage of ghpages)